### PR TITLE
Upgrade tiled and flutter_svg dependencies.

### PR DESCRIPTION
**Pull request by luanpotter**
# Description

Upgrade tiled and flutter_svg dependencies.

Fixes # (issue number, if there is one)

## Type of change

- [x] New feature (non-breaking change which adds functionality)

# Checklist:

If something is unclear, please submit the PR anyways and ask about what you thought was unclear.

- [x] This branch is based on `develop`
- [x] This PR is targeted to merge into `develop` (not `master`)
- [x] I have added an entry under `[next]` in `CHANGELOG.md`
- [x] I have formatted my code with `flutter format`
- [x] I have made corresponding changes to the documentation
- [x] I have added examples for new features in `doc/examples`
- [x] The continuous integration (CI) is passing

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -9,6 +9,7 @@
  - Run linting only on affected and changed examples
  - Add SequenceEffect
  - Fixed bug with travelTime in RotateEffect
+ - Upgrade tiled and flutter_svg dependencies
```
Comment (erickzanardo) [inline CHANGELOG.md:12]: Is develop outdated, check how the CHANGELOG is on master: https://github.com/flame-engine/flame/blob/master/CHANGELOG.md#0220

Comment (luanpotter) [reply CHANGELOG.md:12]: well I'm just adding my line to the changelog, check my diff. If there is a desync between develop and master that is not concern for this PR I think

Comment (luanpotter) [reply CHANGELOG.md]: oh I see my develop was out of date, rebased again. check now. I thought github would compare with current develop

Comment (erickzanardo) [reply CHANGELOG.md]: > I thought github would compare with current develop

Lol, I thought this too

File: .github/workflows/test.yml [hunk 2]
```diff
@@ -13,7 +13,7 @@ jobs:
           java-version: '12.x'
       - uses: subosito/flutter-action@v1
         with:
-          channel: 'stable'
+          channel: 'dev'
```
Comment (erickzanardo) [inline .github/workflows/test.yml:16]: Does that means that people will require the dev channel of Flutter to use Flam?

Comment (luanpotter) [reply .github/workflows/test.yml:16]: No, they will need at least `1.18.0-6.0.pre`. I don't know what branch that is in at the moment. But this is a requirement introduced by flutter_svg 0.18.0, so we can't upgrade if we don't want that.

Comment (erickzanardo) [reply .github/workflows/test.yml:16]: We usually, keep Flame following the beta channel, let me confirm which version is the latest beta.

Comment (luanpotter) [reply .github/workflows/test.yml:16]: I just changed to dev as that's the one I use. I will try beta to see if the build works.

**Pull request merged**